packages/legacy-workbench/src/js/angular/similarity/plugin.js

```diff
@@ -9,7 +9,8 @@ PluginRegistry.add('route', [
         'title': 'view.similarity.indexes.title',
         'helpInfo': 'view.similarity.indexes.helpInfo',
         'documentationUrl': 'semantic-similarity-searches.html#text-based-similarity-searches',
-        'allowAuthorities': ['READ_REPO_{repoId}']
+        'allowAuthorities': ['READ_REPO_{repoId}'],
+        testSelector: 'sub-menu-similarity'
```

Comment:
What is this selector?

Reply:
This is used for selecting the menu and submenus. It was introduced with [MR](https://github.com/Ontotext-AD/graphdb-workbench/pull/2045)